drivers/mlx90393/include/mlx90393_constants.h

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2023 mi6527ri
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_mlx90393
+ * @{
+ *
+ * @file
+ * @brief       MLX90393 internal addresses, registers and constants
+ *
+ * @author      Michael Ristau <michael.ristau@fh-erfurt.de>
+ */
+
+#ifndef MLX90393_CONSTANTS_H
+#define MLX90393_CONSTANTS_H
+
+#ifdef __cplusplus
+extern "C" //{
+#endif
+
+/**
+ * @brief   Register address map
+ */
+#define MLX90393_REG_CONF0          0x00    /**< Configuration register 0 for several parameters **/
+#define MLX90393_REG_CONF1          0x01    /**< Configuration register 1 for several parameters **/
+#define MLX90393_REG_CONF2          0x02    /**< Configuration register 2 for several parameters **/
+#define MLX90393_REG_SENS_TC        0x03    /**< Sensitivity drift compensation factor register **/
+#define MLX90393_REG_OFFSET_X       0x04    /**< Offset register for x axis **/
+#define MLX90393_REG_OFFSET_Y       0x05    /**< Offset register for y axis **/
+#define MLX90393_REG_OFFSET_Z       0x06    /**< Offset register for z axis **/
+#define MLX90393_REG_WOXY_THRESHOLD 0x07    /**< Wake up on change threshold register for x and y axis **/
+#define MLX90393_REG_WOZ_THRESHOLD  0x08    /**< Wake up on change threshold register for z axis **/
+#define MLX90393_REG_WOT_THRESHOLD  0x09    /**< Wake up on change threshold register for temp **/
+#define MLX90393_REG_CONN_TEST      0x0A    /**< Free available register used for connectivity test **/
+#define MLX90393_REG_REF_TEMP       0x24    /**< Reference temperature register **/
```

Comment:
Die Entfernung der überflüssigen Bezeichnung `register` verkürzt die Kommentare
```suggestion
#define MLX90393_REG_CONF1          0x01    /**< Configuration register 1 */
#define MLX90393_REG_CONF2          0x02    /**< Configuration register 2 */
#define MLX90393_REG_SENS_TC        0x03    /**< Sensitivity drift compensation factor  */
#define MLX90393_REG_OFFSET_X       0x04    /**< Offset for x axis */
#define MLX90393_REG_OFFSET_Y       0x05    /**< Offset for y axis */
#define MLX90393_REG_OFFSET_Z       0x06    /**< Offset for z axis */
#define MLX90393_REG_WOXY_THRESHOLD 0x07    /**< Wake up on change threshold for x and y axis **/
#define MLX90393_REG_WOZ_THRESHOLD  0x08    /**< Wake up on change threshold for z axis **/
#define MLX90393_REG_WOT_THRESHOLD  0x09    /**< Wake up on change threshold for temp */
#define MLX90393_REG_CONN_TEST      0x0A    /**< Free available register used for connectivity test **/
#define MLX90393_REG_REF_TEMP       0x24    /**< Reference temperature */
```